pgt/src/PGT.hs

```diff
@@ -53,19 +54,20 @@ data Environment = Environment
 
 type PostProcess = Text -> Text
 
-newtype Config = Config
+data Config = Config
   { clientConfig :: DBT.ClientConfig
+  , setupFile    :: Maybe Path.AbsRelFile
```

Comment:
Are you intentionally limiting to one `setupFile` here, thinking that this one could just as well "source" potential others?